tests/execution-spec-tests/slow.llvm

```diff
@@ -1,2 +1,6 @@
 blockchain_tests/cancun/eip4844_blobs/test_external_vectors.json,*
-blockchain_tests/cancun/eip1153_tstore/test_run_until_out_of_gas.json,*
\ No newline at end of file
+blockchain_tests/cancun/eip1153_tstore/test_run_until_out_of_gas.json,*
+blockchain_tests/osaka/eip7934_block_rlp_limit/test_block_at_rlp_limit_with_logs.json,*
+blockchain_tests/osaka/eip7934_block_rlp_limit/test_block_at_rlp_limit_with_withdrawals.json,*
+blockchain_tests/osaka/eip7934_block_rlp_limit/test_block_rlp_size_at_limit_with_all_typed_transactions.json,*
+blockchain_tests/osaka/eip7934_block_rlp_limit/test_block_at_rlp_size_limit_boundary.json,*
```

Comment:
These are skipped for now. There is a memory leak in the interpreter, causing OOM errors.

<img width="1060" height="136" alt="Image" src="https://github.com/user-attachments/assets/6f820c47-834b-4947-a052-215bd5e49e03" />